### ISSUE TITLE
Build warnings against v13

### ISSUE THREAD

**Issue by devrimgunduz** [laurenz/oracle_fdw#423]
Hi,

I'm seeing these while building oracle_fdw against PG 13. Can you please take a look? Thanks! Devrim

gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fPIC -I/usr/include/oracle/19.6/client64/USE_PGXS=1/sdk/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/oci/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/rdbms/public -I/usr/include/oracle/19.6/client -I/usr/include/oracle/19.6/client64 -I/usr/include/oracle/18.5/client -I/usr/include/oracle/18.5/client64 -I/usr/include/oracle/18.3/client -I/usr/include/oracle/18.3/client64 -I/usr/include/oracle/12.2/client -I/usr/include/oracle/12.2/client64 -I/usr/include/oracle/12.1/client -I/usr/include/oracle/12.1/client64 -I/usr/include/oracle/11.2/client -I/usr/include/oracle/11.2/client64 -I/usr/include/oracle/11.1/client -I/usr/include/oracle/11.1/client64 -I/usr/include/oracle/10.2.0.5/client -I/usr/include/oracle/10.2.0.5/client64 -I/usr/include/oracle/10.2.0.4/client -I/usr/include/oracle/10.2.0.4/client64 -I/usr/include/oracle/10.2.0.3/client -I/usr/include/oracle/10.2.0.3/client64 -I. -I./ -I/usr/pgsql-13/include/server -I/usr/pgsql-13/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include  -c -o oracle_gis.o oracle_gis.c
oracle_gis.c: In function 'setType':
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:695:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(COLLECTION)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:696:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(CIRCSTRING)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:696:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(CIRCSTRING)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:697:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(COMPOUND)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:697:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(COMPOUND)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:698:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(CURVEPOLY)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:698:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(CURVEPOLY)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:699:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(MULTICURVE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:699:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(MULTICURVE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:700:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(MULTISURFACE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:700:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(MULTISURFACE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:701:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(POLYHEDRALSURFACE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:701:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(POLYHEDRALSURFACE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:702:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(TRIANGLE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:702:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(TRIANGLE)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:31: note: here
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                               ^~~~
oracle_gis.c:703:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(TIN)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:694:47: warning: this statement may fall through [-Wimplicit-fallthrough=]
 #define UNSUPPORTED_TYPE( T ) case T ## TYPE: oracleError(FDW_ERROR, "error converting geometry to SDO_GEOMETRY: geometry type "#T" not supported");
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:703:3: note: in expansion of macro 'UNSUPPORTED_TYPE'
   UNSUPPORTED_TYPE(TIN)
   ^~~~~~~~~~~~~~~~
oracle_gis.c:705:3: note: here
   default:
   ^~~~~~~
oracle_gis.c: In function 'ewkbType':
oracle_gis.c:647:4: warning: this statement may fall through [-Wimplicit-fallthrough=]
    oracleError(FDW_ERROR, "error converting SDO_GEOMETRY to geometry: geometry type COLLECTION not supported");
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:648:3: note: here
   case 5:
   ^~~~
oracle_gis.c:655:4: warning: this statement may fall through [-Wimplicit-fallthrough=]
    oracleError(FDW_ERROR, "error converting SDO_GEOMETRY to geometry: geometry type SOLID not supported");
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:656:3: note: here
   case 9:
   ^~~~
oracle_gis.c:657:4: warning: this statement may fall through [-Wimplicit-fallthrough=]
    oracleError(FDW_ERROR, "error converting SDO_GEOMETRY to geometry: geometry type MULTISOLID not supported");
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_gis.c:658:3: note: here
   default:
   ^~~~~~~
/usr/bin/clang -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I/usr/include/oracle/19.6/client64/USE_PGXS=1/sdk/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/oci/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/rdbms/public -I/usr/include/oracle/19.6/client -I/usr/include/oracle/19.6/client64 -I/usr/include/oracle/18.5/client -I/usr/include/oracle/18.5/client64 -I/usr/include/oracle/18.3/client -I/usr/include/oracle/18.3/client64 -I/usr/include/oracle/12.2/client -I/usr/include/oracle/12.2/client64 -I/usr/include/oracle/12.1/client -I/usr/include/oracle/12.1/client64 -I/usr/include/oracle/11.2/client -I/usr/include/oracle/11.2/client64 -I/usr/include/oracle/11.1/client -I/usr/include/oracle/11.1/client64 -I/usr/include/oracle/10.2.0.5/client -I/usr/include/oracle/10.2.0.5/client64 -I/usr/include/oracle/10.2.0.4/client -I/usr/include/oracle/10.2.0.4/client64 -I/usr/include/oracle/10.2.0.3/client -I/usr/include/oracle/10.2.0.3/client64 -I. -I./ -I/usr/pgsql-13/include/server -I/usr/pgsql-13/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o oracle_fdw.bc oracle_fdw.c
oracle_utils.c: In function 'oracleDescribe':
oracle_utils.c:1156:8: warning: this statement may fall through [-Wimplicit-fallthrough=]
     if ((strcmp(type_schema, "SYS") == 0)
        ^
oracle_utils.c:1165:4: note: here
    default:
    ^~~~~~~
/usr/bin/clang -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I/usr/include/oracle/19.6/client64/USE_PGXS=1/sdk/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/oci/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/rdbms/public -I/usr/include/oracle/19.6/client -I/usr/include/oracle/19.6/client64 -I/usr/include/oracle/18.5/client -I/usr/include/oracle/18.5/client64 -I/usr/include/oracle/18.3/client -I/usr/include/oracle/18.3/client64 -I/usr/include/oracle/12.2/client -I/usr/include/oracle/12.2/client64 -I/usr/include/oracle/12.1/client -I/usr/include/oracle/12.1/client64 -I/usr/include/oracle/11.2/client -I/usr/include/oracle/11.2/client64 -I/usr/include/oracle/11.1/client -I/usr/include/oracle/11.1/client64 -I/usr/include/oracle/10.2.0.5/client -I/usr/include/oracle/10.2.0.5/client64 -I/usr/include/oracle/10.2.0.4/client -I/usr/include/oracle/10.2.0.4/client64 -I/usr/include/oracle/10.2.0.3/client -I/usr/include/oracle/10.2.0.3/client64 -I. -I./ -I/usr/pgsql-13/include/server -I/usr/pgsql-13/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o oracle_utils.bc oracle_utils.c
oracle_fdw.c: In function 'oracle_diag':
oracle_fdw.c:671:9: warning: implicit declaration of function 'heap_open'; did you mean 'path_open'? [-Wimplicit-function-declaration]
   rel = heap_open(ForeignServerRelationId, AccessShareLock);
         ^~~~~~~~~
         path_open
oracle_fdw.c:671:7: warning: assignment to 'Relation' {aka 'struct RelationData *'} from 'int' makes pointer from integer without a cast [-Wint-conversion]
   rel = heap_open(ForeignServerRelationId, AccessShareLock);
       ^
oracle_fdw.c:685:3: warning: implicit declaration of function 'heap_close'; did you mean 'heap_delete'? [-Wimplicit-function-declaration]
   heap_close(rel, AccessShareLock);
   ^~~~~~~~~~
   heap_delete
oracle_fdw.c: In function 'oracleGetForeignPlan':
oracle_fdw.c:1182:7: warning: assignment to 'Relation' {aka 'struct RelationData *'} from 'int' makes pointer from integer without a cast [-Wint-conversion]
   rel = heap_open(foreigntableid, NoLock);
       ^
oracle_fdw.c: In function 'oraclePlanForeignModify':
oracle_fdw.c:1653:6: warning: assignment to 'Relation' {aka 'struct RelationData *'} from 'int' makes pointer from integer without a cast [-Wint-conversion]
  rel = heap_open(rte->relid, NoLock);
      ^
oracle_fdw.c: In function 'getColumnData':
oracle_fdw.c:2628:6: warning: assignment to 'Relation' {aka 'struct RelationData *'} from 'int' makes pointer from integer without a cast [-Wint-conversion]
  rel = heap_open(foreigntableid, NoLock);
      ^
In file included from /usr/pgsql-13/include/server/postgres.h:47,
                 from oracle_fdw.c:8:
oracle_fdw.c: In function 'addParam':
/usr/pgsql-13/include/server/utils/elog.h:125:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
  do { \
     ^
/usr/pgsql-13/include/server/utils/elog.h:145:2: note: in expansion of macro 'ereport_domain'
  ereport_domain(elevel, TEXTDOMAIN, __VA_ARGS__)
  ^~~~~~~~~~~~~~
oracle_fdw.c:5608:4: note: in expansion of macro 'ereport'
    ereport(ERROR,
    ^~~~~~~
oracle_fdw.c:5611:3: note: here
   case ORA_TYPE_GEOMETRY:
   ^~~~
/usr/bin/clang -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I/usr/include/oracle/19.6/client64/USE_PGXS=1/sdk/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/oci/include -I/usr/include/oracle/19.6/client64/USE_PGXS=1/rdbms/public -I/usr/include/oracle/19.6/client -I/usr/include/oracle/19.6/client64 -I/usr/include/oracle/18.5/client -I/usr/include/oracle/18.5/client64 -I/usr/include/oracle/18.3/client -I/usr/include/oracle/18.3/client64 -I/usr/include/oracle/12.2/client -I/usr/include/oracle/12.2/client64 -I/usr/include/oracle/12.1/client -I/usr/include/oracle/12.1/client64 -I/usr/include/oracle/11.2/client -I/usr/include/oracle/11.2/client64 -I/usr/include/oracle/11.1/client -I/usr/include/oracle/11.1/client64 -I/usr/include/oracle/10.2.0.5/client -I/usr/include/oracle/10.2.0.5/client64 -I/usr/include/oracle/10.2.0.4/client -I/usr/include/oracle/10.2.0.4/client64 -I/usr/include/oracle/10.2.0.3/client -I/usr/include/oracle/10.2.0.3/client64 -I. -I./ -I/usr/pgsql-13/include/server -I/usr/pgsql-13/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o oracle_gis.bc oracle_gis.c
oracle_fdw.c:671:9: warning: implicit declaration of function 'heap_open' is invalid in C99 [-Wimplicit-function-declaration]
                rel = heap_open(ForeignServerRelationId, AccessShareLock);
                      ^
oracle_fdw.c:671:7: warning: incompatible integer to pointer conversion assigning to 'Relation' (aka 'struct RelationData *') from 'int' [-Wint-conversion]
                rel = heap_open(ForeignServerRelationId, AccessShareLock);
                    ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_fdw.c:685:3: warning: implicit declaration of function 'heap_close' is invalid in C99 [-Wimplicit-function-declaration]
                heap_close(rel, AccessShareLock);
                ^
oracle_fdw.c:1182:9: warning: implicit declaration of function 'heap_open' is invalid in C99 [-Wimplicit-function-declaration]
                rel = heap_open(foreigntableid, NoLock);
                      ^
oracle_fdw.c:1182:7: warning: incompatible integer to pointer conversion assigning to 'Relation' (aka 'struct RelationData *') from 'int' [-Wint-conversion]
                rel = heap_open(foreigntableid, NoLock);
                    ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_fdw.c:1188:3: warning: implicit declaration of function 'heap_close' is invalid in C99 [-Wimplicit-function-declaration]
                heap_close(rel, NoLock);
                ^
oracle_fdw.c:1653:8: warning: implicit declaration of function 'heap_open' is invalid in C99 [-Wimplicit-function-declaration]
        rel = heap_open(rte->relid, NoLock);
              ^
oracle_fdw.c:1653:6: warning: incompatible integer to pointer conversion assigning to 'Relation' (aka 'struct RelationData *') from 'int' [-Wint-conversion]
        rel = heap_open(rte->relid, NoLock);
            ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_fdw.c:1710:2: warning: implicit declaration of function 'heap_close' is invalid in C99 [-Wimplicit-function-declaration]
        heap_close(rel, NoLock);
        ^
oracle_fdw.c:2628:8: warning: implicit declaration of function 'heap_open' is invalid in C99 [-Wimplicit-function-declaration]
        rel = heap_open(foreigntableid, NoLock);
              ^
oracle_fdw.c:2628:6: warning: incompatible integer to pointer conversion assigning to 'Relation' (aka 'struct RelationData *') from 'int' [-Wint-conversion]
        rel = heap_open(foreigntableid, NoLock);
            ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oracle_fdw.c:2675:2: warning: implicit declaration of function 'heap_close' is invalid in C99 [-Wimplicit-function-declaration]
        heap_close(rel, NoLock);
        ^

**Comment by devrimgunduz** [comment]
FWIW, they don't appear against v12. This is RHEL 8 (GCC 8.3.1)

**Comment by laurenz** [comment]
I just tested with the PGDG v13 packages (beta3) on Fedora 32 (gcc 10.2.1), and I cannot see anything like that.

I tried running your first `gcc` invocation, and it worked fine. All I had to change was the Oracle path.

There is something strange in your invocation:

    -I/usr/include/oracle/19.6/client64/USE_PGXS=1/sdk/include

and similar. That looks like you messed up somewhere.

Are you trying to build version 2.2.0 with v13? That will not work, only Git HEAD will do for that.
But it would certainly not fail with weird messages like that for `oracle_gis.c`.

I'll release 2.3.0 or 3.0.0. soon (before v13 GA hopefully) which should work fine with v13.
